test/functional/blockchain.py

```diff
@@ -34,14 +37,26 @@ def __init__(self):
         super().__init__()
         self.setup_clean_chain = False
         self.num_nodes = 1
+        self.extra_args = [['-stopatheight=207']]
 
     def run_test(self):
         self._test_getchaintxstats()
         self._test_gettxoutsetinfo()
         self._test_getblockheader()
         self._test_getdifficulty()
         self._test_getnetworkhashps()
-        self.nodes[0].verifychain(4, 0)
+        self._test_stopatheight()
+        assert self.nodes[0].verifychain(4, 0)
+
+    def _test_stopatheight(self):
```

Comment:
micronit: The other subtests are defined in the order that they're called. Perhaps place this definition after the other subtests?